.github/workflows/ci.yml

```diff
@@ -282,11 +282,12 @@ jobs:
     strategy:
       matrix:
         checks:
-          - bans 
-          - sources
+          - bans sources
+          - advisories
+          - licenses
 
     # Prevent sudden announcement of a new advisory from failing ci:
-    continue-on-error: ${{ matrix.checks == 'advisories' }}
+    continue-on-error: ${{ matrix.checks == 'advisories' || 'licenses'}}
```

Comment:
We need to decide how we want to handle security and license warnings.

Here are my suggestions:
- warnings should block merging, so we don't have confusing failure icons on all our PRs

Warnings in existing dependencies:
- advisories: ask for a security review, open a ticket, then disable the advisory
- license warnings: make a list and give it to Dodger to check the licenses are allowed

Warnings in new dependencies:
- choose another dependency, or
- follow the process for existing dependencies

We also need to work out who will handle the warnings in existing dependencies. (Because they aren't related to the actual PR that they are failing on.)

Can you add this discussion to the next engineering sync meeting?
(I'll mark this PR as draft until we make a decision.)

Reply:
@gustavovalverde do you want to create a discussion to decide this?

Reply:
Can you open two separate discussions, please:
- security advisories
- supported licences